articles/libraries/chemistry/concepts/symmetries-molecular.md

```diff
@@ -63,16 +66,30 @@ In addition to enumerating over all orbital integrals that are numerically ident
 
 Rather than constructing a Fermionic Hamiltonian by adding `FermionTerm`s, we may directly add all terms corresponding to each orbital integral automatically. For example, the following code automatically enumerates over all permutational symmetries and orders the terms in canonical order: 
 ```csharp
-    // We create a `FermionHamiltonian` object to store the `FermionTerm`
-    // instances.
-    var hamiltonian = new FermionHamiltonian();
+    // We load the namespace containing fermion objects. This
+    // example also uses LINQ queries.
+    using Microsoft.Quantum.Chemistry.Fermion;
+    using System.Linq;
 
     // We create a `OrbitalIntegral` object to store a two-electron molecular 
     // orbital integral data.
     var orbitalIntegral = new OrbitalIntegral(new[] { 0, 1, 2, 3 }, 0.123);
 
-    // We now add all `FermionTerm`s corresponding to the orbital integral. 
-    // This automatically enumerates over all symmetries, and orders 
-    // the `FermionTerm` instances in canonical order.
-    hamiltonian.AddFermionTerm(orbitalIntegral);
+    // We create an `OrbitalIntegralHamiltonian` object to store the orbital integral
+    // terms
+    var orbitalIntegralHamiltonian = new OrbitalIntegralHamiltonian();
+    orbitalIntegralHamiltonian.Add(orbitalIntegral);
+
+    // We now convert the orbital integral representation to a fermion
+    // representation. This also requires choosing a convention for 
+    // mapping spin orbital indices to integer indices.
+    var fermionHamiltonian = orbitalIntegralHamiltonian.ToFermionHamiltonian(IndexConvention.UpDown);
+
+    // Alternatively, we we add orbital integrals directly to a fermion Hamiltonian
+    // as follows. This automatically enumerates over all symmetries, and then
+    // orders the `HermitianFermionTerm` instances in canonical order. We will need to
```

Comment:
Here as in the above, it would be better to use passive voice and avoid "We". Such as "An indexing convention etc."